doc/_static/page_history.js

```
@@ -326,11 +270,19 @@ function get_titles(data, k) {
-function get_vdata(data, v) {
+
+function get_vdata(data, v, file, lang) {
+    info = '';
+    link = '';
     for (var i = 0; i < data.length; i++) {
-        if (data[i].v == v) { return data[i]; }
+        if (v <= data[i].v) { info = data[i]; break; }
     }
-    return '';
+    if (info === '') {
+        link = lang + '/' + file + '.html">' + v + '</a>';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)